services/events/serverless.yml

```diff
@@ -17,7 +18,8 @@ provider:
   cfLogs: true
   environment:
     ENVIRONMENT: ${file(../../config.${self:provider.stage}.json):ENVIRONMENT}
-  
+    NODE_ENV: ${env:NODE_ENV}
```

Comment:
@voctory added this to ensure local DDB is used when running locally. basically the env variable wasnt picked up when running npm run dev, but we need to let serverless know it has these env vars.